docs/sysvinit/meson.build

```diff
@@ -1,6 +1,8 @@
-file = configure_file(
-        input : 'README.in',
-        output : 'README',
-        configuration : substs)
-install_data(file,
-             install_dir : sysvinit_path)
+if conf.get('HAVE_SYSV_COMPAT', 0) == 1
+        file = configure_file(
+                input : 'README.in',
+                output : 'README',
+                configuration : substs)
+        install_data(file,
+                     install_dir : sysvinit_path)
```

Comment:
Hm, so what do you think about just applying the conditional to `install_data` alone, i.e. building almost everything unconditionally? The build is fast enough that the extra build time does not matter, but it might allow us to catch more mistakes early.